src/test/java/org/folio/rest/impl/VoucherLinesApiTest.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.rest.impl;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+import org.folio.rest.jaxrs.model.Errors;
+import org.folio.rest.jaxrs.model.VoucherLine;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+
+import static org.folio.rest.RestVerticle.OKAPI_HEADER_TENANT;
+import static org.folio.rest.impl.AbstractHelper.ID;
+import static org.junit.Assert.assertEquals;
+
+
+public class VoucherLinesApiTest extends ApiTestBase {
+
+  private static final Logger logger = LoggerFactory.getLogger(VoucherLinesApiTest.class);
+
+  static final Header NON_EXIST_CONFIG_X_OKAPI_TENANT = new Header(OKAPI_HEADER_TENANT, "invoicetest");
```

Comment:
looks like this variable is not being used anywhere?

Reply:
Yes, removed it.